pipelines/azure-ci.yml

```diff
@@ -1,25 +1,33 @@
-# Test TF2-GNN package
+# Test TF2-GNN package and build sdist to be consumed by release pipeline
 trigger:
   branches:
     include:
-      - '*'  # must quote since "*" is a YAML reserved character; we want a string
```

Comment:
Maybe the Github diff interface isn't super-helpful here, but it looks like publish will get triggered off other branches as well. Am I missing something?

Reply:
Publish just leaves an artifact in the pipeline-associated storage, upload to PyPI still requires a manual trigger in Azure DevOps interface.